pkg/controller/serviceaccount/tokens_controller.go

```diff
@@ -57,19 +62,53 @@ type TokensControllerOptions struct {
 	// SecretResync is the time.Duration at which to fully re-list secrets.
 	// If zero, re-list will be delayed as long as possible
 	SecretResync time.Duration
-	// This CA will be added in the secretes of service accounts
+	// This CA will be added in the secrets of service accounts
 	RootCA []byte
+
+	// RetryQPS controls the rate at which failed syncs are retried
+	// If zero, a default QPS is used
+	RetryQPS float64
+	// RetryBurst controls the rate at which failed syncs are retried
+	// If zero, a default burst is used
+	RetryBurst int64
```

Comment:
Is there any good reason to have a burst here other than 1? Just trying to make the options simpler